src/common/BulletList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function BulletList({ children, listType = 'bullet_circle' }) {
+  return <ListContainer listType={listType}>{children}</ListContainer>
+}
+
+const ListContainer = styled.ul`
+  padding: 0;
+  margin: 0 0 30px 0;
+
+  li {
+    list-style: none;
+    background: ${(props) =>
+      'url(/img/' + props.listType + '.svg) no-repeat left top;'};
+    padding: 0px 0px 3px 27px;
```

Comment:
Seems to be very accurate. Have you tried it on different screen sizes and mobile sizes?

Reply:
Yes, I tested it on different screen sizes. This controls the positioning of the bullet icon (circle or checkmark). I want a fixed distance on all screen sizes.